server/backend/courses/migrations/0002_course_image_alter_rating_rating.py

```
@@ -0,0 +1,28 @@
+            model_name="course",
+            name="image",
+            field=models.URLField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name="rating",
+            name="rating",
+            field=models.PositiveSmallIntegerField(
+                validators=[
+                    django.core.validators.MinValueValidator(1),
+                    django.core.validators.MaxValueValidator(5),
+                ]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)